include/emp/web/Div.hpp

```diff
@@ -52,6 +57,8 @@ namespace web {
     class TableInfo;
     class DivInfo : public internal::WidgetInfo {
       friend Element; friend Div; friend TableInfo;
+      friend prefab::ButtonGroup;
```

Comment:
Why do ButtonGroup and ControlPanel need to be friends? What are they needing to access inside of Div???

Reply:
They need to be able to construct a `DivInfo` instance but the constructor is protected.

Reply:
How did we get around this with other prefab components? It would be best to try to preserve levilization if possible (although these predeclares aren’t a /huge/ issue).